code/datums/elements/noticable_organ.dm

```diff
@@ -0,0 +1,48 @@
+/**
+ * noticable organ element; which makes organs have a special description added to the person with the organ, if certain body zones aren't covered.
```

Comment:
shouldnt red eyes from cult/being high use this?

Reply:
neither are from organs!